src/configmanager.cpp

```diff
@@ -234,6 +234,7 @@ bool ConfigManager::load()
 	boolean[REMOVE_WEAPON_AMMO] = getGlobalBoolean(L, "removeWeaponAmmunition", true);
 	boolean[REMOVE_WEAPON_CHARGES] = getGlobalBoolean(L, "removeWeaponCharges", true);
 	boolean[REMOVE_POTION_CHARGES] = getGlobalBoolean(L, "removeChargesFromPotions", true);
+	boolean[PZLOCK_SKULLED_ATTACKER] = getGlobalBoolean(L, "pzLockSkulledAttacker", true);
```

Comment:
Sorry for being picky, but change name to pzLockSkullAttacker, skulled isn't exactly right, mb for suggesting it